frontend/appflowy_flutter/packages/flowy_infra_ui/lib/src/flowy_overlay/flowy_dialog.dart

```diff
@@ -25,14 +25,14 @@ class FlowyDialog extends StatelessWidget {
     final windowSize = MediaQuery.of(context).size;
     final size = windowSize * 0.7;
     return SimpleDialog(
+        contentPadding: EdgeInsets.zero,
         title: title,
         shape: shape ??
             RoundedRectangleBorder(borderRadius: BorderRadius.circular(8)),
         children: [
           Material(
             type: MaterialType.transparency,
             child: Container(
-              padding: padding,
```

Comment:
Why do you delete this line?

Reply:
To not have the Snackbar have horizontal padding, but to fill the length of the dialog.

I've substituted it further down in the tree.

Reply:
I can also make it have the horizontal padding, and make the snackbar rounded, we need a design for snackbar anyway, so we can change it then. I wasn't able to find it at least.